recipes/asyncinotify/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "asyncinotify" %}
+{% set version = "2.0.2" %}
+{% set blakesha256 = "36deda93616582b1806883b69e2d79c2ad967b14a1e893383933f20788d0d569" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/{{ blakesha256[0:2] }}/{{ blakesha256[2:4] }}/{{ blakesha256[4:] }}/{{ name }}-{{ version }}-py3-none-any.whl
```

Comment:
This needs to be built from the sources and not a wheel. As the upstream git repository doesn't contain tags and `sdist`s aren't available on PyPI you'll have to ask upstream to do one (or both).